content/03.results.md

```diff
@@ -287,7 +287,7 @@ Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of
 In contrast, LGGs showed significant downregulation (Bonferroni-corrected p < 0.05) of these pathways.
 Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].
 Indeed, we observed significant upregulation of these cytokines in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and found immune cell types dominated by monocytes in these tumors (**Figure {@fig:Fig5}C**).
-We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in LGGs and craniopharyngiomas compared to medulloblastoma and ependymoma tumors (Bonferroni-corrected p < 0.05), both of which showed significant down-regulation of these cytokines (**Figure {@fig:Fig5}B**).
+We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in both LGGs and craniopharyngiomas when compared to either medulloblastoma or ependymoma tumors (Bonferroni-corrected p < 0.05) which had significant down-regulation of these cytokines (**Figure {@fig:Fig5}B**).
```

Comment:
This aspect "which had significant down-regulation of these cytokines" I think is still a little ambiguous as to which histology it refers to.
But, since the sentence starts with upregulation, maybe we can just remove the end phrase as it's implied?

Reply:
yeah good point

Reply:
updated this